### PR TITLE
Create initial example app

### PR DESCRIPTION
**Pull request by emmawillis**
https://github.com/DevCycleHQ-Sandbox/example-javascript/assets/25067948/2fe744a9-72f1-48ff-8a9d-a6d2ddd65d64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+
+An example app built using the [DevCycle JavaScript SDK](https://docs.devcycle.com/sdk/client-side-sdks/javascript/)
+
+## Running the example
+
+* Run `yarn install` in the project directory to install dependencies
+* Create a `.env` file and set `DEVCYCLE_CLIENT_SDK_KEY` to the SDK Key for your environment.\
+You can find this under [Settings > Environments](https://app.devcycle.com/r/environments) on the DevCycle dashboard.
+* [Create a new feature](https://docs.devcycle.com/introduction/quickstart#2-create-a-feature) on the dashboard with a string variable named `greeting` and a boolean variable named `spin-logo`
+* Open `index.html` to view it in your browser.
```
Comment (laurawarr) [inline README.md:29]: they'll probably need to run `yarn build` before this step?

Comment (emmawillis) [reply README.md:29]: oops good catch

File: src/devcycle.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const devcycleOptions = { logLevel: "debug" };
+const devcycleClient = initializeDevCycle(
+    process.env.DEVCYCLE_CLIENT_SDK_KEY,
+    user,
+    devcycleOptions
+);
+
+
+devcycleClient.onClientInitialized().then(({ config }) => {
+    console.log("DevCycle client initialized with configuration: ", config);
```
Comment (laurawarr) [inline src/devcycle.js]: do we actually want this log? the sdk seems to be fairly verbose with the logging on init

**Review by laurawarr — APPROVED** [review]

**Comment by laurawarr** [comment]
Oh, it'd be nice if we could add a simple test to verify things are working as expected

File: src/devcycle.js (hunk 2 above)
Comment (emmawillis) [reply src/devcycle.js]: true, i'll drop it

**Review by laurawarr — APPROVED** [review]

**Pull request merged**